index.js

```diff
@@ -10,11 +10,10 @@ const md5 = require("md5");
 const isString = require("lodash.isstring");
 const cloneDeep = require("lodash.clonedeep");
 const version = require("./package.json").version;
-
```

Comment:
We don't need version any more do we?

Reply:
It's sent along in the data
![Screen Shot 2022-09-15 at 9 10 02 AM](https://user-images.githubusercontent.com/1200974/190454216-d208cfd2-927f-4660-9fae-6bff4c33e646.png)